src/Server/Test/Unit/Services/Scp/InstanceStoredNotificationServiceTest.cs

```diff
@@ -74,6 +74,7 @@ public void NewInstanceStored_NoSupportedObservers()
             observer.Verify(p => p.OnNext(It.IsAny<InstanceStorageInfo>()), Times.Once());
             _cleanupQueue.Verify(p => p.QueueInstance(instance.InstanceStorageFullPath), Times.Once());
 
+            cancel.Dispose();
```

Comment:
duplicate line?

Reply:
Did it on purpose to make sure it doesn't crash.